catroid/src/main/java/org/catrobat/catroid/common/LookData.java

```diff
@@ -209,6 +209,13 @@ public String getLookFileName() {
 		return fileName;
 	}
 
+	public void setLookRgf(byte[] rgf) {
```

Comment:
i would not put this in `LookData` since it is specific to `LegoImageLookData`
Rather have it in `LegoImageLookData` without override and cast usages to `LegoImageLookData` wherever needed